src/tree/CosmosDBTreeItem.ts

```diff
@@ -24,6 +24,7 @@ export class CosmosDBTreeItem extends AzureParentTreeItem<ISiteTreeRoot> {
     public readonly parent!: SiteTreeItem;
     public readonly client: IAppSettingsClient;
     public cosmosDBExtension: vscode.Extension<AzureExtensionApiProvider | undefined> | undefined;
+    public suppressMaskLabel = true;
```

Comment:
Doesn't the `client` properties above require to be added to `valuesToMask`? 
Something similar to...
`const valuesToMask = [ client.siteName, client.slotName, client.defaultHostName, client.resourceGroup,
            client.planName, client.planResourceGroup, client.kuduHostName, client.gitUrl,
            site.repositorySiteName, ...(site.hostNames || []), ...(site.enabledHostNames || [])
        ];`

Reply:
The app settings client? I don't think so. Since this tree item is a child of the SiteTreeItem, it will inherit all of the same valuesToMask. See here where I loop up through the tree based on `parent`: https://github.com/microsoft/vscode-azuretools/blob/main/ui/src/tree/addTreeItemValuesToMask.ts#L17

If you have specific properties in mind that you think I should add, though, let me know

Reply:
Never mind, I was thinking more of the appSettings keys and values. But I found that they are taken care of in the appservice package.

Reply:
I'm actually not sure the stuff in the appservice package will cover every case, but I also can't do the same thing I'm doing with the `SiteClient` example you mentioned. `SiteClient` already has a bunch of values we can mask, but `IAppSettingsClient` requires a separate async call to list the app settings. I may investigate app settings stuff more, but for now I think this PR is a big step in the right direction so I'll go ahead and merge as-is.